### PR TITLE
[L1T] fixes to l1t module names

### PR DESCRIPTION
**Pull request by cecilecaillol**
#### PR description:

Fix L1 collection names after general renaming in earlier PR. 

@srimanob FYI

### CODE REVIEW

**Comment by cecilecaillol** [comment]
please test

File: L1Trigger/Configuration/python/SimL1Emulator_cff.py [hunk 1]
```diff
@@ -167,7 +167,7 @@
 _phase2_siml1emulator.add(l1tTrackJetsExtendedEmulation)
 
 from L1Trigger.L1TTrackMatch.l1tTrackerEmuEtMiss_cfi import *
-l1tTrackerEmuEtMiss.L1VertexInputTag = ("L1VertexFinderEmulator","l1verticesEmulation")
+l1tTrackerEmuEtMiss.L1VertexInputTag = ("l1tVertexFinderEmulator","l1verticesEmulation")
```
Comment (srimanob) [inline L1Trigger/Configuration/python/SimL1Emulator_cff.py:170]: Is this line the same as in https://github.com/cms-sw/cmssw/pull/39401, this line of change:
https://github.com/cms-sw/cmssw/pull/39401/files#diff-c0b64ac0c0be97495049537672bca975f5be0de08e90cb72bd6000e8496bbf4cL170

**Comment by srimanob** [comment]
Thanks very much @cecilecaillol 
I've only a comment of a line which already propose the change in https://github.com/cms-sw/cmssw/pull/39401.

**Comment by cecilecaillol** [comment]
> Thanks very much @cecilecaillol I've only a comment of a line which already propose the change in #39401.

Yes it is the same fix, I guess it is ok to have in both PRs since the change is identical

**Comment by cecilecaillol** [comment]
+l1

**Comment by srimanob** [comment]
+Upgrade

This PR fixes the remaining L1 collection names `L1VertexFinderEmulator` ==> `l1tVertexFinderEmulator`. For one change which is the same as [39401](https://github.com/cms-sw/cmssw/pull/39401/files#diff-c0b64ac0c0be97495049537672bca975f5be0de08e90cb72bd6000e8496bbf4c), I would ask suggestion from @perrotta @rappoccio

**Comment by perrotta** [comment]
+1

**Comment by missirol** [comment]
type bugfix

Will this PR be backported to 12_5_X ?

**Comment by epalencia** [comment]
Hi @missirol , changes in this PR are included in this backport https://github.com/cms-sw/cmssw/pull/39451

**Comment by missirol** [comment]
Ah okay, thanks @epalencia .

**Pull request merged**